examples/esi_example_variants/src/main.rs

```diff
@@ -0,0 +1,123 @@
+use std::{
+    collections::{HashMap, VecDeque},
+    io::Write,
+};
```

Comment:
No need to use `VecDeque`
```suggestion
use std::{collections::HashMap, io::Write};
};
```